benchexec/tablegenerator/__init__.py

```diff
@@ -1382,10 +1386,201 @@ def write_head_line(
         out.write("\n")
 
 
+def write_tex_command_table(
+    out,
+    run_sets: List[RunSetResult],
+    stats: List[List[ColumnStatistics]],
+    **kwargs,
+):
+    bench_name_set = set()
+    for benchmark in run_sets:
+        bench_name_formatted = LatexCommand.format_command_part(
+            benchmark.attributes.get("benchmarkname")
+        )
+        if bench_name_formatted in bench_name_set:
+            logging.error(
+                "Duplicated formatted benchmark name %s detected. Benchmark names must be unique for Latex"
+                "\nSkipping writing to file %s"
+                % (bench_name_formatted, kwargs["title"] + ".tex")
+            )
+            return
+        bench_name_set.add(bench_name_formatted)
+
+    header = textwrap.dedent(
+        """\
+        % The following definition defines a command for each value.
+        % The command name is the concatenation of the first six arguments.
+        % To override this definition, define \\StoreBenchExecResult with \\newcommand before including this file.
+        % Arguments: benchmark name, run-set name, category, status, column name, statistic, value
```

Comment:
I still need to think about whether we should add more arguments here, for cases where users overwrite `StoreBenchExecResult` with their own command and could make use of it.